atom/browser/api/atom_api_window.cc

```diff
@@ -631,6 +631,10 @@ bool Window::HasShadow() {
   return window_->HasShadow();
 }
 
+void Window::SetOpacity(const double opacity) {
```

Comment:
For consistency with other API's we should probably have a `GetOpacity` as well 👍